tests/data_examples/targets/payload.json

```diff
@@ -19,5 +19,7 @@
             },
             "path": "file2.tar.gz"
         }
-    ]
+    ],
+    "publish_targets": true,
+    "add_task_id_to_custom": false
```

Comment:
I think we should remove the optional fields from the example `payload.json`.
It will force us to test the default value scenario. If someone removes the field as Optional or changes the default we can notice it in the tests.

Reply:
I added it in the example because otherwise, I had to change multiple tests and add a value to those fields because using `pydantic.Field()` for them with a default value means they will always have a value no matter if we give them inside the `payload.json`.
So if we don't want them in this example I would have to add them manually in the payload in the tests.